lightspeed-stack.template.yaml

```diff
@@ -19,3 +19,6 @@ user_data_collection:
   transcripts_storage: "/tmp/data/transcripts"
 authentication:
   module: "noop-with-token"
+customization:
+  system_prompt_path: "/tmp/systemprompt.txt"
+  disable_query_system_prompt: true
```

Comment:
I think we should have this as false in the dev repo and true in prod, so you can still play around with the prompt without redeploying